src/models/session/mocks/flow-with-internal-portals.ts

```diff
@@ -0,0 +1,104 @@
+import { ComponentType, FlowGraph } from "../../../types";
+
+export const flow: FlowGraph = {
+  _root: {
+    edges: ["kReDM5AWwf", "levelOne", "DJDsTYVeZG"],
+  },
+  DJDsTYVeZG: {
+    data: {
+      fn: "animal",
+      tags: [],
+      text: "Elephant",
+      neverAutoAnswer: false,
+    },
+    type: ComponentType.Question,
+    edges: ["LNjFPmokA4"],
+  },
```

Comment:
nit: appreciate that you've updated to human-readable `type` here, but still a bit hard to read tests referencing random node IDs - eg `DJDsTYVeZG` &rarr; `levelOneQuestionTwo` would help me ! (Even if just for node IDs that show up in `expect()` statements, no need to rename `Answer` IDs necessarily ! )

Reply:
No this is a very fair and good comment thanks - changed in [145a49b](https://github.com/theopensystemslab/planx-core/pull/553/commits/145a49b23a9238c0c6cec70b2dc4bddec8857a97)